### DISCUSSION TITLE
Img2Img Alternative Test

### DISCUSSION THREAD

**Discussion by Nargodian** [WASasquatch/was-node-suite-comfyui#31] [Ideas]
I suggested it in ComfyUI's discussions but it got buried, But then WASasquatch was the only person to actually reply so perhaps you might give it a shot.

Its a method of rather then just comping noise into the image, you work the image through the denoiser somehow to generate the noise, with the idea that for sequences of images would have similar noise patterns, this beats out freezing the noise since that just makes it feel like their is dirt on the lens.

In A1111's webui its a script called img2imgalt.py And my limited python makes it seem reasonable.

I'm kinda sure that with this one extra step on top of the batching that this node pack added you could get Corridor Digital to adopt it, they love them node workflows.

**Comment by WASasquatch** [comment]
I'll see if I can get this in: https://gist.github.com/trygvebw/c71334dd127d537a15e9d59790f7f5e1

![image](https://user-images.githubusercontent.com/1151589/230224100-02e8633f-4e62-4912-82fc-1a27e7c3abba.png)

It doesn't have all the convoluted stuff for webui and stuff, and already or k sampler, so probably easier to implement. No promises though. I tend to break tensors left and right. :P

**Reply by Nargodian** [reply]
Nice!

**Comment by AceroMagico** [comment]
> I suggested it in ComfyUI's discussions but it got buried

To put it mildly I would say that foresight is not the first quality in ComfyUI's discussions out of here... ;)
